src/CoreManager.ts

```diff
@@ -343,7 +343,7 @@ const config: Config & Record<string, any> = {
   SERVER_AUTH_TOKEN: null,
   LIVEQUERY_SERVER_URL: null,
   ENCRYPTED_KEY: null,
-  VERSION: 'js' + require('../package.json').version,
+  VERSION: 'js' + process.env.npm_package_version,
```

Comment:
I believe this is injected by npm, so it only works when running a script via npm. So it's not the same as getting the version from the package.json file. Is this limitation acceptable?

Reply:
It gets injected when you build the project

`dist/parse.js`
<img width="334" alt="Screenshot 2025-04-14 at 12 35 23 PM" src="https://github.com/user-attachments/assets/3fae162f-d598-4331-8b6b-235939fc82fe" />

`lib/node/CoreManager.js`
<img width="308" alt="Screenshot 2025-04-14 at 12 36 34 PM" src="https://github.com/user-attachments/assets/f77b8e2b-528d-45cd-9c22-7060ef48d19e" />

Reply:
Alright